tests/scipy_stats_test.py

```diff
@@ -174,7 +174,12 @@ def args_maker():
       x, loc, scale = map(rng, shapes, dtypes)
       return [x, loc, onp.abs(scale)]
 
+    def args_maker2():
+      _, loc, scale = map(rng, shapes, dtypes)
+      return [0.5, loc, onp.abs(scale)]
+
     self._CheckAgainstNumpy(scipy_fun, lax_fun, args_maker, check_dtypes=True)
+    self._CheckAgainstNumpy(scipy_fun, lax_fun, args_maker2, check_dtypes=True)
```

Comment:
Is the intent to test a scalar here, as opposed to an array that has 0 dimensions? It turns out there's a magic value you can pass as the shape to get the RNG to produce a Numpy scalar:
https://github.com/google/jax/blob/master/tests/lax_numpy_test.py#L44

Does that fulfill your needs?

Reply:
Sorry, my bad. I have interpreted the wrapper genNamedParametersNArgs wrongly. My intend was to make the shapes for `x`, `loc`, `scale` differently but `genNamedParametersNArgs` already did it. So the test which I added was redundant and did not capture the error in uniform implementation. I have replaced it with a new test which explicitly captures the shape error.